lib/rouge/lexers/brainfuck.rb

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Brainfuck < RegexLexer
+      tag 'brainfuck'
+      filenames '*.b', '*.bf'
+      mimetypes 'text/x-brainfuck'
+
+      title "Brainfuck"
+      desc "The Brainfuck programming language"
+
+      # optional comment or whitespace
+      ws = %r((?:\s|[^><\[\]\+\-\.,]*)+)
+      id = /[a-zA-Z_][a-zA-Z0-9_]*/
+
+      def self.keywords
```

Comment:
self.keywords, self.keywords_type, self.reserved, self.builtins should not be needed here, since you don't make use of them anywhere. I'm assuming you've started with another lexer, but the RegexLexer superclass does not dictate any of this.